tunneling/python/app.py

```diff
@@ -0,0 +1,35 @@
+import json
+import urllib
+
+import flask
+import requests
+from flask_cors import CORS
+
+app = flask.Flask(__name__)
+CORS(app)
+
+
+@app.route("/bugs", methods=["POST"])
+def bugs():
+    sentry_host = "sentry.io"
+    known_project_ids = ["123456"]
+
+    envelope = flask.request.data.decode("utf-8")
+    piece = envelope.split("\n")[0]
+    header = json.loads(piece)
+    dns = urllib.parse.urlparse(header.get("dsn"))
+
+    if dns.hostname != sentry_host:
+        print("Invalid sentry host")
+        return {}
+
+    project_id = dns.path.strip("/")
+    if project_id not in known_project_ids:
+        print(f"Invalid project id: {project_id}")
+        return {}
+
+    url = f"https://{sentry_host}/api/{project_id}/envelope/"
```

Comment:
This made me realize that this example is Python3-only (and that's okay) -- could you please add a README file similar to https://github.com/getsentry/examples/tree/master/tunneling/dotnet and https://github.com/getsentry/examples/tree/master/tunneling/nextjs?